mne/beamformer/_compute_beamformer.py

```diff
@@ -173,12 +194,15 @@ def _normalized_weights(Wk, Gk, Cm_inv_sq, reduce_rank, nn, sk):
     Wk_max = np.matmul(max_power_ori[:, np.newaxis], Wk)[:, 0]
     Gk_max = np.matmul(Gk, max_power_ori[:, :, np.newaxis])
     denom = np.matmul(Gk_max.transpose(0, 2, 1),
-                      np.matmul(Cm_inv_sq[np.newaxis], Gk_max))[:, 0]
+                      np.matmul(Cm_inv[np.newaxis], Gk_max))[:, 0]
     np.sqrt(denom, out=denom)
     Wk_max /= denom
+
     # All three entries get the same value from this operation
     Wk[:] = Wk_max[:, np.newaxis]
 
+    return Wk
+
 
 @contextmanager
```

Comment:
@larsoner I missed that PR back then (I was deep in grant writing land) - what does this do, out of curiosity? Why do you add a null contextmanager here?

Reply:
> Why do you add a null contextmanager here?

Purely for ease of review of a PR. I removed an indentation level but it would have been difficult to tell what was actually changed if I unindented everything. I meant to remove it but forgot.

We could remove it in the renaming aesthetic PR suggested above

Reply:
That makes sense, thanks for the explanation! Let's take care of it in the renaming PR.